webapp/WEB-INF/templates/admin/plugins/forms/entries/fill_entry_type_date.html

```diff
@@ -46,6 +46,7 @@
 	</#list>
 </#if>
 <#-- <@getDatePickerBootstrap idField="${idName}" language=locale.language /> -->
+<script type="text/javascript" src="js/jquery/plugins/moment.min.js"></script>
```

Comment:
Is it related to this PR ?
we should generally avoid adding a dependency on a generic JS library for specific needs: we should study the possibility of using only native JS, or integrating the necessary functionality into a shared library at the Forms or Lutece-core level